From d613a67b081df2b90fd4f16fe992a22f6947efe8 Mon Sep 17 00:00:00 2001
From: QbProg <tholag@gmail.com>
Date: Sun, 24 Jul 2011 18:05:40 +0200
Subject: [PATCH 04/11] [warning-fix][missing-operators]

Added placement delete operators with allocators, when there is
new() method.
---
 inc/Poly_CoherentTriPtr.hxx |    6 ++++++
 inc/TDF_LabelNode.hxx       |    6 ++++++
 2 files changed, 12 insertions(+), 0 deletions(-)

Index: oce/inc/Poly_CoherentTriPtr.hxx
===================================================================
--- oce.orig/inc/Poly_CoherentTriPtr.hxx
+++ oce/inc/Poly_CoherentTriPtr.hxx
@@ -82,6 +82,11 @@ class Poly_CoherentTriPtr
   // ---------- PUBLIC METHODS ----------
 
 
+  void operator delete (void * aBuffer, const Handle(NCollection_BaseAllocator)& theAllocator) 
+  {
+    theAllocator->Free(aBuffer);
+  };
+
   /**
    * Constructor.
    */
